system/Helpers/text_helper.php

```diff
@@ -851,3 +851,27 @@ function excerpt(string $text, string $phrase = null, int $radius = 100, string
 
 	//--------------------------------------------------------------------
 }
+
+if (!function_exists('str_contains')) {
+    /**
+     * Determine if a given string contains in a given substring.
+     *
+     * @param  string  $haystack
+     * @param  string|string[]  $needles
+     * @return bool
+     */
+    function str_contains(string $haystack, string $needles): bool
+    {
+        if (phpversion() >= 8) {
```

Comment:
Is this ever reachable? If so, it would be an infinite loop, I think.

Reply:
Maybe I get this wrong, but if we are at PHP > 8, then `str_contains` will already exist so our helper will not be defined. Thus  there should be no need to check the version again.